lib/winrm-elevated/scripts/elevated_shell.ps1

```diff
@@ -94,6 +94,9 @@ do {
   $err_cur_line = SlurpOutput $err_file $err_cur_line 'err'
 } while (!($registered_task.state -eq 3))
 
+del $out_file
```

Comment:
@mwrock maybe we can remove the logs in normal circumstances, but do something for debugging by setting an environment variable on the client side which results in uploading a modified script that does not remove the logs? Your mileage may vary....